packages/fx-core/src/core/middleware/utils/v3MigrationUtils.ts

```diff
@@ -80,3 +88,24 @@ export async function getProjectVersion(ctx: CoreHookContext): Promise<string> {
   }
   return "0.0.0";
 }
+
+export async function fsReadAndConvertUserdata(
```

Comment:
Can we just call it `readAndConvertUserdata`?

Reply:
sure